keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -551,16 +600,22 @@ Below are some examples to consider, in addition to the aforementioned [maturity
 -->
 #### Alpha
 
-- Feature implemented behind a feature flag
+- Feature implemented behind a feature gate
 - Integration tests added for all relevant verbs (POST, PUT, and PATCH)
 
-<!--
 #### Beta
 
-- Gather feedback from developers and surveys
-- Complete features A, B, C
-- Additional tests are in Testgrid and linked in KEP
+- [  ]kubectl validate flag offers ability to perform server-side validation
+- [ ] endpoints handler unit testing of field validation
+- [ ] customresource handler unit testing of field validation
+- [ ] field validation integration tests check for exact match of strict errors
```

Comment:
what do you mean by exact match? exact match relative to existing client-side errors, or exact match of expected errors (instead of substring matches)?

Reply:
just exact match of expected errors, because right now we're just checking that the error from the server contains each expected error because strict errors are reported as one large combined error rather than individual errors like for warnings. 

See https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/field_validation_test.go#L540-L557